src/main/webapp/pages/analyses/analyses.html

```diff
@@ -9,6 +9,7 @@
     <link rel="stylesheet" th:href="@{/dist/css/analyses.bundle.css}" />
     <script th:inline="javascript">
       window.PAGE = {
+        canManage: /*[[${isAdmin}]]*/ true,
```

Comment:
Shouldn't the default be set to false rather than true?

Reply:
This isn't a default or anything.  It is just there, kind of zen like.